salesagility/SuiteCRM
# Gitattributes + codeception.dist.yml

## Description



- Added a .gitattributes file to handle LF vs CRLF line endings and to prevent certain files when creating archives.
- normalised all of the remaining CRLF file endings in the repo and changed them to LF.
- Added a renamed codeception.yml to codeception.dist.yml as per the [docs](https://codeception.com/docs/reference/Configuration).

## How To Test This

No real way to test, I guess you could pull down the repo and check for any CRLF files I may have missed. The codeception file rename will kick off in Travis.

## Types of changes


### Final checklist
